tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -300,6 +300,22 @@ var CESIUM_MAP_VIEW = (function() {
         m_viewer.trackedEntity = entity;
     }
 
+    // Set Cesium primitives
+    cesium_load_primitives = function()
+    {
+        var $map_element = $('#' + m_map_target);
+        var m_primitives_options = [];
+        var raw_primitives_options = $map_element.data('primitives');
```

Comment:
Is `raw_primitive_options` supposed to be saved as `m_primitive_options` or do we store the processed options? Look at `m_entities_options` and `m_models_options` to see how they are used and follow the same pattern.